futures-util/src/future/future/mod.rs

```diff
@@ -27,7 +26,7 @@ mod fuse;
 mod map;
 
 delegate_all!(
-    /// Future for the [`flatten`](super::FutureExt::flatten) method.
```

Comment:
Please split unrelated changes into separate PRs as they will be difficult to review.